klasses/bootcamp_admissions_client_test.py

```diff
@@ -140,11 +142,26 @@ def test_json_raises(test_data, mocked_get_200):
 
 def test_can_pay_klass(test_data, mocked_get_200):
     """
-    Test BootcampAdmissionClient.can_pay_klass
+    Test BootcampAdmissionClient.can_pay_klass with a mix of legacy and fluidreview klasses
     """
     user, _ = test_data
+    WebhookRequestFactory(award_id=19, user_email=user.email, user_id=user.profile.fluidreview_id)
 
     boot_client = BootcampAdmissionClient(user)
     assert boot_client.can_pay_klass(16) is True
```

Comment:
Can you make variables for `12` `16` near the sample data and `19` locally?

Reply:
✅